client/my-sites/domains/domain-management/dns/mx-record.jsx

```diff
@@ -14,56 +15,73 @@ import FormTextInput from 'components/forms/form-text-input';
 import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
 import FormSettingExplanation from 'components/forms/form-setting-explanation' ;
 
-const MxRecord = React.createClass( {
-	statics: {
-		initialFields: {
-			name: '',
-			data: '',
-			aux: 10
-		}
-	},
+class MxRecord extends React.PureComponent {
+	static initialFields = {
+		name: '',
+		data: '',
+		aux: 10
+	};
 
-	propTypes: {
+	static propTypes = {
 		fieldValues: React.PropTypes.object.isRequired,
 		onChange: React.PropTypes.func.isRequired,
 		selectedDomainName: React.PropTypes.string.isRequired,
 		show: React.PropTypes.bool.isRequired
-	},
-
-	fieldNames: [ 'name', 'data', 'aux' ],
+	};
 
 	render() {
-		const classes = classnames( { 'is-hidden': ! this.props.show } ),
-			isValid = this.props.isValid,
-			isNameValid = isValid( 'name' ),
-			isDataValid = isValid( 'data' ),
-			isAuxValid = isValid( 'aux' );
+		const { fieldValues, isValid, onChange, selectedDomainName, show, translate } = this.props;
+		const classes = classnames( { 'is-hidden': ! show } );
+		const isNameValid = isValid( 'name' );
+		const isDataValid = isValid( 'data' );
+		const isAuxValid = isValid( 'aux' );
 
 		return (
 			<div className={ classes }>
 				<FormFieldset>
-					<FormLabel>{ this.translate( 'Name', { context: 'Dns Record' } ) }</FormLabel>
+					<FormLabel>
+						{ translate( 'Name', { context: 'Dns Record' } ) }
+					</FormLabel>
 					<FormTextInputWithAffixes
 						name="name"
-						placeholder={ this.translate( 'Enter subdomain (optional)', { context: 'Placeholder shown when entering the optional subdomain part of a new DNS record' } ) }
+						placeholder={
+							translate(
+								'Enter subdomain (optional)',
```

Comment:
No idea why it thinks so. It's the same, just added a line break. :man_shrugging: